src/main/java/com/eerussianguy/blazemap/feature/atlas/AtlasTask.java

```diff
@@ -107,6 +110,11 @@ void exportAsync() {
             File file = getExportFile();
             file.getParentFile().mkdirs();
             image.writeToFile(file);
+
+            // Completed Stage: Send the filepath into chat
+            setStage(Stage.COMPLETE);
+            String saveMessage = String.format("Atlas save to: %s", file.getPath());
+            Helpers.getPlayer().sendMessage(new TextComponent(saveMessage), Util.NIL_UUID);
```

Comment:
This should be a translated component with a key in the lang file.
I know mine don't all have this yet, but those are still a work in progress.
You go do yours now :p